core/common/src/main/java/alluxio/conf/Source.java

```diff
@@ -52,6 +53,10 @@
      * HadoopConf). This source has the highest priority.
      */
     RUNTIME,
+    /**
+     * The property value is specified as options for a mount point.
+     */
+    MOUNT_OPTION,
```

Comment:
are we intentional to have mount option with higher priority than the runtime ?

Reply:
This is consistent with the present behavior. I think we should keep it that way. I’ll update the comment above.

Reply:
make sense